SupportClient/SupportClient/Controllers/FeedbackViewController.swift

```diff
@@ -0,0 +1,219 @@
+//
+//  FeedbackViewController.swift
+//  SupportClient
+//
+//  Created by Cameron Russell on 3/6/20.
+//  Copyright © 2020 Cornell Appdev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+import DZNEmptyDataSet
+import BSImagePicker
+import Photos
+
+class FeedbackViewController: UIViewController {
+
+    private var attachedErrorFiles = [PHAsset]()
+    private var collectionView: UICollectionView!
+    
+    private let messageLabel = UILabel()
+    private let messageTextField = UITextField()
+    private let typeLabel = UILabel()
+            
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = UIColor.backgroundColor
+        setupNavigationBar()
+        setupCollectionView()
+        setupConstraints()
+    }
+    
+    func setupNavigationBar() {
+        self.navigationController?.navigationBar.barTintColor = UIColor.navigationTintColor
+        let attributes = [
+            NSAttributedString.Key.font: UIFont._21RobotoMedium!,
+            NSAttributedString.Key.foregroundColor: UIColor.titleColor
+        ]
+        let attributedTitle = NSAttributedString(string: "New Feedback", attributes: attributes)
+        title = attributedTitle.string
+        navigationItem.leftBarButtonItem = UIBarButtonItem(
+            title: "Cancel",
+            style: UIBarButtonItem.Style.plain,
+            target: self,
+            action: #selector(handleNavigationBarLeftTap)
+        )
+        navigationItem.rightBarButtonItem = UIBarButtonItem(
+            title: "Send",
+            style: UIBarButtonItem.Style.plain,
+            target: self,
+            action: #selector(handleNavigationBarRightTap)
+        )
+    }
+    
+    func setupCollectionView() {
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .horizontal
+        collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        collectionView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
note: u don't need to do this w snapkit but since we're gonna switch its fine to keep

Reply:
`collectionView` currently has NSLayoutConstraints not SnapKit constraints, so needed